03.bowling/bowling.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'debug'
+
+class Array
+  def strike?
+    return true if self[0] == 10
+
+    false
+  end
+
+  def spare?
+    return true if sum == 10
+
+    false
+  end
```

Comment:
こちらも
```suggestion
  def spare?
    sum == 10
 end
```

で十分です。

`==` 自体が true か falseを返します。